README
NAME
    Task::BeLike::RSRCHBOY - Modules RSRCHBOY uses!

VERSION
    version 0.003

TASK CONTENTS
  CPAN/GitHub Tools
   App::cpanminus
   App::cpanoutdated
   App::gh
   Dist::Zilla 4
   Dist::Zilla::PluginBundle::RSRCHBOY
   Git::CPAN::Patch
  DB bits
   DBD::SQLite 1.31
   DBIx::Class 0.08192
   DBIx::Class::Schema::Loader 0.07010
   DBIx::Class::Migration
  Date/Time handling
   DateTime
   DateTime::Format::DB2
   DateTime::Format::DBI
   DateTime::Format::MySQL
   DateTime::Format::Pg
  Hygiene
   common::sense
   namespace::autoclean
   Perl::Tidy
   Perl::Critic
  Class bits
   Class::Method::Modifiers
   Role::Basic
   Reindeer 0.008
   MooseX::App::Cmd
   MooseX::AutoDestruct 0.007
   MooseX::TrackDirty::Attributes 2.000
    Version 2.000 required because: native traits support

   MooseX::Types::Perl
  Testing
   Test::Moose::More 0.009
   Test::Routine 0.015
  Other Libraries I Use
   aliased
   opts 0.05
   Capture::Tiny
   Config::JFDI
   File::chdir
   File::Slurp 9999.18
   Path::Class
   Smart::Args
   Smart::Comments
   Sub::Install
   Sub::Exporter
   Try::Tiny
   V
SEE ALSO
    Task::BeLike::RSRCHBOY::Web

AUTHOR
    Chris Weyl <cweyl@alumni.drew.edu>

COPYRIGHT AND LICENSE
    This software is Copyright (c) 2011 by Chris Weyl.

    This is free software, licensed under:

      The GNU Lesser General Public License, Version 2.1, February 1999



Changes
Revision history for Task-BeLike-RSRCHBOY

0.003     2012-05-29 14:51:31 America/Los_Angeles
- misc version updates; add Software::License::CCpack

0.002     2012-03-30 20:45:53 America/Los_Angeles
- Split web deps out

0.001     2011-09-29 11:23:39 America/Los_Angeles


lib/Task/BeLike/RSRCHBOY.pm
#
# This file is part of Task-BeLike-RSRCHBOY
#
# This software is Copyright (c) 2011 by Chris Weyl.
#
# This is free software, licensed under:
#
#   The GNU Lesser General Public License, Version 2.1, February 1999
#
package Task::BeLike::RSRCHBOY;
{
  $Task::BeLike::RSRCHBOY::VERSION = '0.003';
}

# ABSTRACT: Modules RSRCHBOY uses!

!!42;



=pod

=head1 NAME

Task::BeLike::RSRCHBOY - Modules RSRCHBOY uses!

=head1 VERSION

version 0.003

=head1 TASK CONTENTS

=head2 CPAN/GitHub Tools

=head3 L<App::cpanminus>

=head3 L<App::cpanoutdated>

=head3 L<App::gh>

=head3 L<Dist::Zilla> 4

=head3 L<Dist::Zilla::PluginBundle::RSRCHBOY>

=head3 L<Git::CPAN::Patch>

=head2 DB bits

=head3 L<DBD::SQLite> 1.31

=head3 L<DBIx::Class> 0.08192

=head3 L<DBIx::Class::Schema::Loader> 0.07010

=head3 L<DBIx::Class::Migration>

=head2 Date/Time handling

=head3 L<DateTime>

=head3 L<DateTime::Format::DB2>

=head3 L<DateTime::Format::DBI>

=head3 L<DateTime::Format::MySQL>

=head3 L<DateTime::Format::Pg>

=head2 Hygiene

=head3 L<common::sense>

=head3 L<namespace::autoclean>

=head3 L<Perl::Tidy>

=head3 L<Perl::Critic>

=head2 Class bits

=head3 L<Class::Method::Modifiers>

=head3 L<Role::Basic>

=head3 L<Reindeer> 0.008

=head3 L<MooseX::App::Cmd>

=head3 L<MooseX::AutoDestruct> 0.007

=head3 L<MooseX::TrackDirty::Attributes> 2.000

Version 2.000 required because: native traits support

=head3 L<MooseX::Types::Perl>

=head2 Testing

=head3 L<Test::Moose::More> 0.009

=head3 L<Test::Routine> 0.015

=head2 Other Libraries I Use

=head3 L<aliased>

=head3 L<opts> 0.05

=head3 L<Capture::Tiny>

=head3 L<Config::JFDI>

=head3 L<File::chdir>

=head3 L<File::Slurp> 9999.18

=head3 L<Path::Class>

=head3 L<Smart::Args>

=head3 L<Smart::Comments>

=head3 L<Sub::Install>

=head3 L<Sub::Exporter>

=head3 L<Try::Tiny>

=head3 L<V>

=head1 SEE ALSO

L<Task::BeLike::RSRCHBOY::Web>

=head1 AUTHOR

Chris Weyl <cweyl@alumni.drew.edu>

=head1 COPYRIGHT AND LICENSE

This software is Copyright (c) 2011 by Chris Weyl.

This is free software, licensed under:

  The GNU Lesser General Public License, Version 2.1, February 1999

=cut


__END__



Makefile.PL

use strict;
use warnings;

use 5.006;

use ExtUtils::MakeMaker 6.30;



my %WriteMakefileArgs = (
  "ABSTRACT" => "Modules RSRCHBOY uses!",
  "AUTHOR" => "Chris Weyl <cweyl\@alumni.drew.edu>",
  "BUILD_REQUIRES" => {
    "File::Find" => 0,
    "File::Temp" => 0,
    "Test::More" => "0.88",
    "strict" => 0,
    "warnings" => 0
  },
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => "6.30"
  },
  "DISTNAME" => "Task-BeLike-RSRCHBOY",
  "EXE_FILES" => [],
  "LICENSE" => "lgpl",
  "NAME" => "Task::BeLike::RSRCHBOY",
  "PREREQ_PM" => {
    "App::cpanminus" => 0,
    "App::cpanoutdated" => 0,
    "App::gh" => 0,
    "Capture::Tiny" => 0,
    "Class::Method::Modifiers" => 0,
    "Config::JFDI" => 0,
    "DBD::SQLite" => "1.31",
    "DBIx::Class" => "0.08192",
    "DBIx::Class::Migration" => 0,
    "DBIx::Class::Schema::Loader" => "0.07010",
    "DateTime" => 0,
    "DateTime::Format::DB2" => 0,
    "DateTime::Format::DBI" => 0,
    "DateTime::Format::MySQL" => 0,
    "DateTime::Format::Pg" => 0,
    "Dist::Zilla" => 4,
    "Dist::Zilla::PluginBundle::RSRCHBOY" => 0,
    "File::Slurp" => "9999.18",
    "File::chdir" => 0,
    "Git::CPAN::Patch" => 0,
    "MooseX::App::Cmd" => 0,
    "MooseX::AutoDestruct" => "0.007",
    "MooseX::TrackDirty::Attributes" => "2.000",
    "MooseX::Types::Perl" => 0,
    "Path::Class" => 0,
    "Perl::Critic" => 0,
    "Perl::Tidy" => 0,
    "Reindeer" => "0.008",
    "Role::Basic" => 0,
    "Smart::Args" => 0,
    "Smart::Comments" => 0,
    "Sub::Exporter" => 0,
    "Sub::Install" => 0,
    "Test::Moose::More" => "0.009",
    "Test::Routine" => "0.015",
    "Try::Tiny" => 0,
    "V" => 0,
    "aliased" => 0,
    "common::sense" => 0,
    "namespace::autoclean" => 0,
    "opts" => "0.05"
  },
  "VERSION" => "0.003",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



